src/client.ts

```diff
@@ -233,7 +233,7 @@ export default class Client {
       this.cluster = decoded.clusterIdentifier || '1';
     } catch (error) {
       this.failure = true;
-      console.error('Error while authenticating, err: ', error);
+      console.error('Error while authenticating, err: ' + JSON.stringify(error));
```

Comment:
Passing it as its original value should means it's inspectable in a console as well as serialised in logs. The failure of the display in the GCP console is more of a problem with their logging, not an issue with our SDK. I'd move this from a bug ticket to a chore or story and bump the version as a minor rather than a patch. This isn't a bug